core/src/main/java/com/graphhopper/routing/VirtualEdgeIteratorState.java

```diff
@@ -143,7 +144,40 @@ public EdgeIteratorState setName( String name )
         this.name = name;
         return this;
     }
+    
+    @Override
+    public boolean getBoolean(int key, boolean _default, PMap kwargs )
+    {
+        if (key == EdgeIteratorState.DISPREFERED_STARTSTOPEDGE)
```

Comment:
public constant variables should get better names and be shorter and/or better readable -> INFERIOR_START_STOP_EDGE or just INFERIOR_EDGE_KEY?